### ISSUE TITLE
adapt-contrib-matching: NVDA not reading combo box items

### ISSUE THREAD

**Issue by canstudios-nicolaw** [adaptlearning/adapt_framework#2464]
To note the matching question on which this was noticed also had trickle waiting for it's completion. When the combo box is opened using the keyboard controls the list options are not read out when you arrow through them. Only after selection is the chosen option read out. Navigation then stalls here. It seems stuck and you cannot arrow down. The screen reader just says 'blank'. You can use tab back down to the next question option but tabbing starts from the top of the page.

### Your environment
* 0.8.1 AT & 4.2.0 FW
* IE11
* Windows 10
* Matching 3.0.0

### Steps to reproduce
- Create a course with a matching component and navigate through it with NVDA on IE11
